fixtures/http_pass.yaml

```diff
@@ -4,4 +4,10 @@ http:
     thresholdMillis: 3000
     responseCodes: [201,200,301]
     responseContent: ""
-    maxSSLExpiry: 60
\ No newline at end of file
+    maxSSLExpiry: 60
+icmp:
```

Comment:
I think remove this from the http_pass file 
+ add a failing icmp check

Reply:
@moshloop removed. icmp_fail.yaml added